proxies/nodejs/app.ts

```diff
@@ -21,6 +22,22 @@ const handleClient = async (ctx) => {
   }
 }
 
+const handleUser = async (ctx) => {
+  const user: DVCUser = ctx.request.body
+
+  const userId = Object.keys(data.users).length
+  data.users[userId] = user
+
+  ctx.status = 200
+  ctx.set("Location", `user/${userId}`)
+  ctx.body = {
+    entityType: "User",
```

Comment:
should we be enumerating the entity types somewhere as a type?

Reply:
good call